addons/l10n_es/data/account_tax_group_data.xml

```diff
@@ -101,5 +101,13 @@
             <field name="name">Retenciones 35%</field>
             <field name="country_id" ref="base.es"/>
         </record>
+        <record id="tax_group_irnr_no_ue" model="account.tax.group">
+            <field name="name">Retenciones no residentes no-UE</field>
```

Comment:
Why do not we put them in the same tax groups as the once with the percentages?  Tax groups are mainly according to percentages...  They will deduce from elsewhere that it is for no residentes no-UE.

Reply:
I put new taxes in the same tax groups as the once with the percentages, but I had to create new tax group *Retenciones 0%*